src/supermarket/app/controllers/collaborators_controller.rb

```diff
@@ -40,6 +40,10 @@ def create
 
       add_group_members_as_collaborators(resource, collaborator_params[:group_ids]) if collaborator_params[:group_ids].present?
 
+      if collaborator_params[:resourceable_type] == 'Cookbook'
```

Comment:
@nellshamrell @robbkidd can I get a 👍  on not changing this or are there opinions that I should change it to `resource.class == 'Cookbook'`? Definitely willing to do either way.